generic/rcutorture.py

```diff
@@ -39,16 +40,13 @@ def setUp(self):
         """
         self.results = []
         self.log.info("Check if CONFIG_RCU_TORTURE_TEST is enabled\n")
-        ret = os.system(
-            'cat /boot/config* | grep CONFIG_RCU_TORTURE_TEST=m')
-        if ret != 0:
+        ret = linux_modules.check_kernel_config('CONFIG_RCU_TORTURE_TEST')
+        if ret == 0:
```

Comment:
can `if  ret == linux_modules.MODULE` use instead of `if ret == 0` as  linux_modules having a MODULE global variable .